app/uk/gov/hmrc/gform/sharedmodel/FrontEndSubmissionDesIncludeIfEval.scala

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.gform.sharedmodel
+
+import play.api.libs.json.{ Format, Json }
+import uk.gov.hmrc.gform.sharedmodel.formtemplate.destinations.DestinationId
+
+case class FrontEndSubmissionDesIncludeIfEval(exprEval: List[(DestinationId, Boolean)])
```

Comment:
here as well, I am not sure FrontEndSubmission adds anything to the understanding of purpose of the class

Reply:
I thought the name should align with others. Look at `FrontEndSubmissionVariables` and `FrontEndSubmissionVariablesBuilder`

Reply:
What `..Des..` in `FrontEndSubmissionDesIncludeIfEval` stands for? Is it shorthand for `Destination` or does it refers to DES service?

Looking at all the places this is used I tend to assume `Des` is shorthand for `Destination`. Can we use `Dest` in these cases (or full `Destination`) here and in other places?